.github/workflows/front-office-qa.yml

```diff
@@ -0,0 +1,20 @@
+name: Static code analysis assets
+on:
+    pull_request:
+        branches:
+            - dev
+        paths:
+            - '**.ts'
+            - '**.tsx'
+            - '**.md'
+            - '**.scss'
+            - 'front-office-qa.yml'
+jobs:
+    wp-scripts-lint:
+        uses: inpsyde/reusable-workflows/.github/workflows/wp-scripts-lint.yml@main
+        secrets:
+            GITHUB_USER_EMAIL: ${{ secrets.DEPLOYBOT_EMAIL }}
+            GITHUB_USER_NAME: ${{ secrets.DEPLOYBOT_USER }}
```

Comment:
I'm not sure where exactly these are used, as far as I can see there is no git operations in the workflow, does it make sense to omit them, they are optional inputs anyway.